en/docs/install-and-setup/deploy-with-docker.md

```
@@ -68,7 +68,18 @@ These resources have been created based on the most common WSO2 Open Banking dep
 This steps builds the container image for WSO2 Identity Server with WSO2 Open Banking Identity and Access Management Accelerator Module
 
 1. Navigate to `<OB_DOCKER_HOME>/samples/wso2is_with_obiam` directory.
-2. Execute `docker build` command as shown below.
+2. Log in to the WSO2 Docker registry.
+
+    ```shell
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)